.travis.yml

```diff
@@ -6,8 +6,6 @@ addons:
     - python-numpy
     - python-matplotlib
     - python-scipy
-    - python-pandas
-    - python-seaborn
```

Comment:
How about installing an old version of seaborn? You could do "pip install seaborn==0.5"